### ISSUE TITLE
show empty sky if no tile is requested when switching between spatially matched images

### ISSUE THREAD

**Issue by kswang1029** [CARTAvis/carta-frontend#819]
in the video, three spatially matched images can be seen with the initial zoom level. However when we zoom in to a certain scale, one image will be outside the view box so no tiles are requested. Since no tile is delivered to the frontend, we still see previous image, not an empty sky. 
![Mar-06-2020 14-31-05](https://user-images.githubusercontent.com/20819712/76058357-2d301080-5fb7-11ea-8ee8-a43f8e7bac56.gif)

**Comment by arusstaylor** [comment]
but isn't empty sky the right thing to do, since that is the correct matched part of the sky.

**Comment by veggiesaurus** [comment]
> but isn't empty sky the right thing to do, since that is the correct matched part of the sky.

Yes. The issue is it doesn't re-render the empty sky, and the old image persists

**Comment by arusstaylor** [comment]
I see.  Yes that is a problem.

**Comment by Jordatious** [comment]
I also have this problem, which happens quite a bit since I always spatially match by default. I'm currently imaging small boxes around outlier sources so it happens as I switch between those and the larger image.

**Comment by kswang1029** [comment]
This seems fixed when the multi-panel view was implemented. So closed.
